src/features/components/atoms/tab/types.ts

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { TFlexDirection } from 'features/types'
+import { TString } from 'types/generics'
+
+export type TTabItem = {
+    title: TString
+    component?: React.ReactElement
+    image: React.ReactElement
+    alt?: string
+}
+
+export type TStepperTab = {
+    items: TTabItem[]
```

Comment:
@NikitK-deriv I guess @mohsen-deriv said not to use `T`. Please confirm from him. If i am wrong then ignore my comments 🙏

Reply:
Let me watch master-class videos 🙏 
I'm still busy for that